src/main/java/com/example/spring/dto/book/UpdateBookRequestDto.java

```diff
@@ -0,0 +1,27 @@
+package com.example.spring.dto.book;
+
+import com.example.spring.validate.annotations.CoverImage;
+import com.example.spring.validate.annotations.Isbn;
+import jakarta.annotation.Nullable;
+import jakarta.validation.constraints.Min;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+public class UpdateBookRequestDto {
+    @Nullable

```

Comment:
```suggestion
    @NotBlank
```

check all places with String

Reply:
I need the string to be possibly null. That`s why i used @Nullable. Probably it is better to remove  this annotion at all.